sessions/react-state-3/journal-app-favorite-button-shared-state/src/components/Badge/Badge.css

```diff
@@ -0,0 +1,14 @@
+.badge {
+  display: inline-block;
+  font-size: 12px;
```

Comment:
`px` is used as unit all across the code. Changing it would be a lot of would in six challenges in two PRs :D  

It's not wrong to use `px` for font-size. Using a relative unit is favorable in regards of the base font-size set in the browser so one can increase the font-size independent from zooming the whole layout. Using `px` doesn't prevent increasing the font-size so I think it's okay to use it here.